### ISSUE TITLE
No option is available for changing characters' Style

### ISSUE THREAD

**Issue by Vazkii** [jedmund/hensei-web#253]
Currently only relevant for Yngwie (water), who has an alternate style, but might be more of an issue in the future

https://gbf.wiki/Yngwie

**Comment by jedmund** [comment]
Yeah, this is an intentional omission. Since the only character is Yngwie, I decided I had bigger fish to fry, but I will get around to this eventually! It's actually pretty easy, I'm just lazy.

**Comment by jedmund** [comment]
When are they going to add more styles? Cygames???
